src/transformers/pipelines/__init__.py

```diff
@@ -528,8 +528,11 @@ def pipeline(
     # Use default model/config/tokenizer for the task if no model is provided
     if model is None:
         # At that point framework might still be undetermined
-        model = get_default_model(targeted_task, framework, task_options)
-        logger.warning(f"No model was supplied, defaulted to {model} (https://huggingface.co/{model})")
+        model, default_revision = get_default_model_and_revision(targeted_task, framework, task_options)
+        revision = revision if revision is not None else default_revision
+        logger.warning(
+            f"No model was supplied, defaulted to {model} and revision {revision} (https://huggingface.co/{model})"
```

Comment:
Can we use `HF_ENDPOINT` variable for `https://huggingface.co` ?

Reply:
yes agreed